shared/src/commonMain/kotlin/at/asitplus/wallet/app/common/KeystoreProvider.kt

```diff
@@ -37,7 +39,16 @@ class KeystoreService(
     private suspend fun initSigner(): KeyWithSelfSignedCert {
         getProvider().let { provider ->
             val forKey = provider.getSignerForKey(Configuration.KS_ALIAS).getOrElse {
-                provider.createSigningKey(alias = Configuration.KS_ALIAS).getOrThrow()
+                provider.createSigningKey(alias = Configuration.KS_ALIAS, configure = {
```

Comment:
this here: if you just upgrade from a previous version, I don't think you'll get biometry. And since iOS does not delete keys on uninstall, I am concerned about inconsistent and/or unexpected behaviour.

Reply:
Okay, so what would be a solution here? New key would invalidate all existing credentials.

Reply:
Let's do this in https://github.com/a-sit-plus/valera/issues/203, I've removed that commit from this PR